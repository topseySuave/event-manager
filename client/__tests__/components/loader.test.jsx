import React from 'react';
import { mount, shallow } from 'enzyme';
import { CircularLoader, IndeterminateLoader } from
  '../../src/components/loader';

describe('Loader Component', () => {
  const circularLoader = shallow(<CircularLoader />);
  const indeterminateLoader = shallow(<IndeterminateLoader />);
  test('should make a snapshot of the CircularLoader component', () => {
    expect(circularLoader).toMatchSnapshot();
  });

  test('should make a snapshot of the IndeterminateLoader component', () => {
    expect(indeterminateLoader).toMatchSnapshot();
  });
});
