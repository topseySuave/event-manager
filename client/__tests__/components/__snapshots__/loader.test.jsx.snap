// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component make a snapshot of the CircularLoader component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CircularLoader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="preloader-wrapper small active"
      >
        <div
          className="spinner-layer spinner-blue"
        >
          <div
            className="circle-clipper left"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="gap-patch"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="circle-clipper right"
          >
            <div
              className="circle"
            />
          </div>
        </div>
        <div
          className="spinner-layer spinner-red"
        >
          <div
            className="circle-clipper left"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="gap-patch"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="circle-clipper right"
          >
            <div
              className="circle"
            />
          </div>
        </div>
        <div
          className="spinner-layer spinner-yellow"
        >
          <div
            className="circle-clipper left"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="gap-patch"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="circle-clipper right"
          >
            <div
              className="circle"
            />
          </div>
        </div>
        <div
          className="spinner-layer spinner-green"
        >
          <div
            className="circle-clipper left"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="gap-patch"
          >
            <div
              className="circle"
            />
          </div>
          <div
            className="circle-clipper right"
          >
            <div
              className="circle"
            />
          </div>
        </div>
      </div>,
      "className": "center-align",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="spinner-layer spinner-blue"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>,
          <div
            className="spinner-layer spinner-red"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>,
          <div
            className="spinner-layer spinner-yellow"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>,
          <div
            className="spinner-layer spinner-green"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>,
        ],
        "className": "preloader-wrapper small active",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>,
            ],
            "className": "spinner-layer spinner-blue",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper left",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "gap-patch",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper right",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>,
            ],
            "className": "spinner-layer spinner-red",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper left",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "gap-patch",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper right",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>,
            ],
            "className": "spinner-layer spinner-yellow",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper left",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "gap-patch",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper right",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>,
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>,
            ],
            "className": "spinner-layer spinner-green",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper left",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "gap-patch",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="circle"
                />,
                "className": "circle-clipper right",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "circle",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="preloader-wrapper small active"
        >
          <div
            className="spinner-layer spinner-blue"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>
          <div
            className="spinner-layer spinner-red"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>
          <div
            className="spinner-layer spinner-yellow"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>
          <div
            className="spinner-layer spinner-green"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>
        </div>,
        "className": "center-align",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="spinner-layer spinner-blue"
            >
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>
            </div>,
            <div
              className="spinner-layer spinner-red"
            >
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>
            </div>,
            <div
              className="spinner-layer spinner-yellow"
            >
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>
            </div>,
            <div
              className="spinner-layer spinner-green"
            >
              <div
                className="circle-clipper left"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="gap-patch"
              >
                <div
                  className="circle"
                />
              </div>
              <div
                className="circle-clipper right"
              >
                <div
                  className="circle"
                />
              </div>
            </div>,
          ],
          "className": "preloader-wrapper small active",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="circle-clipper left"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="gap-patch"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="circle-clipper right"
                >
                  <div
                    className="circle"
                  />
                </div>,
              ],
              "className": "spinner-layer spinner-blue",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper left",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "gap-patch",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper right",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="circle-clipper left"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="gap-patch"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="circle-clipper right"
                >
                  <div
                    className="circle"
                  />
                </div>,
              ],
              "className": "spinner-layer spinner-red",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper left",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "gap-patch",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper right",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="circle-clipper left"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="gap-patch"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="circle-clipper right"
                >
                  <div
                    className="circle"
                  />
                </div>,
              ],
              "className": "spinner-layer spinner-yellow",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper left",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "gap-patch",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper right",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="circle-clipper left"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="gap-patch"
                >
                  <div
                    className="circle"
                  />
                </div>,
                <div
                  className="circle-clipper right"
                >
                  <div
                    className="circle"
                  />
                </div>,
              ],
              "className": "spinner-layer spinner-green",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper left",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "gap-patch",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="circle"
                  />,
                  "className": "circle-clipper right",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "circle",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Loader Component make a snapshot of the IndeterminateLoader component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IndeterminateLoader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="indeterminate gradient__bg"
      />,
      "className": "progress",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "indeterminate gradient__bg",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="indeterminate gradient__bg"
        />,
        "className": "progress",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "indeterminate gradient__bg",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
